### PR TITLE
Add hero and strapline cta fields

### PR DESCRIPTION
**Pull request by kevinhowbrook**

### CODE REVIEW

File: rca/home/models.py [hunk 1]
```diff
@@ -12,18 +15,71 @@ class HomePage(BasePage):
...
+        get_image_model_string(),
         null=True,
+        blank=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    hero_colour_option = models.CharField(
+        max_length=1,
+        choices=(
+            ("1", "Light text on a dark image"),
```
Comment (tm-kn) [inline rca/home/models.py]: If we are going to use numbers, why not just use `PositiveSmallIntegerField`?

**Comment by kevinhowbrook** [comment]
Closing in favour of https://github.com/torchbox/rca-wagtail-2019/pull/119

File: rca/home/models.py (hunk 1 above)
Comment (kevinhowbrook) [reply rca/home/models.py]: I attempted this but it resulted in `django.db.utils.DataError: invalid input syntax for integer: ""`

Comment (tm-kn) [reply rca/home/models.py]: Did you change `"1"` to `1`?

**Pull request closed without merging**